toolkits/springer/packages/springer-hero/demo/dist/index.html

```diff
@@ -1728,7 +1728,7 @@
 
 .c-hero__image {
   position: relative;
-  min-height: 20rem;
```

Comment:
get the content right first, rather than get the CSS to make sure it's the right size ✊🏼